the-blue-alliance-ios/View Elements/Match/MatchViewModel.swift

```diff
@@ -44,15 +47,53 @@ struct MatchViewModel {
             }
             return true
         }()
-
+        
```

Comment:
Let's get rid of these please. Sorry - I don't have a linter or anything. I am our linter.